### PR TITLE
Updated Requirements

### LINKED ISSUE #4: Version bump for the dependencies in requirement list
This should be easy. Run a requirement check, and update the list with newer versions of dependencies.

### PR DESCRIPTION
**Pull request by rexdivakar**
Existing requirements has been updated,
PyQt5==5.14.0 & PyQt5-sip==12.7.2 has been integrated towards PyQt5==5.15.1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-PyQt5==5.14.0
-PyQt5-sip==12.7.2
```
Comment (gridhead) [inline requirements.txt:2]: You don't seem to have updated the `PyQt5-sip`.

**Review by gridhead — CHANGES_REQUESTED** [review]
Please make the changes suggested in the comments.

**Review by rexdivakar — COMMENTED** [review]
My Linux machine didn't have the need for PyQt5-sip so i have skipped it but added it now pls validate and approve @t0xic0der

**Review by gridhead — APPROVED** [review]
Looks good to me.

**Pull request merged**